developers/guides/getstarted/setup-local-development.md

```diff
@@ -34,6 +36,14 @@ deploying Solana programs to the blockchain.
 - how to create a Solana wallet for developing
 - how to airdrop SOL tokens for your wallet
 
+## Windows users only
+
+Users on Windows 10 or 11 should install:
+
+- [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/install), which provides a Linux environment that runs inside your existing Windows installation. You can then run regular Linux software, including the Linux versions of Solana CLI. 
+- [Windows Terminal](https://www.microsoft.com/store/productId/9N0DX20HK701), a modern terminal for Windows.
+
+Once you've installed both Windows Subsystem for Linux and Windows Terminal, you can continue with this guide.
```

Comment:
Can you add a brief section/details on how to actually connect to WSL which will then be used to run the shell commands?

Reply:
Sure, done!